src/pages/assets/AssetsPage.js

```diff
@@ -34,10 +34,9 @@ const AssetsPage = props => (
       <link rel='canonical' href={`${getOrigin()}/assets`} />
     </Helmet>
     {props.isBetaModeEnabled && (
-      <WidgetList
-        listName={getHeadTitle(props.type, props.location.search)}
+      <GetTotalMarketcap
         type={props.type}
-        isLoggedIn={props.isLoggedIn}
+        listName={getHeadTitle(props.type, props.location.search)}
```

Comment:
@DmitriVanGuard Maybe better to use another name for getHeadTitle function? Also you can send one argument in this function "props"